### DISCUSSION TITLE
Tasti per posizione rapida tenda da sole

### DISCUSSION THREAD

**Discussion by ilmaurino** [indomus/forum#414] [HUB: Home Assistant]
Buongiorno a tutti.

Ho uno shelly 2.5 che mi controlla una tenda da sole. è connesso MQTT e correttamente calibrato.
Ho la card che mi da le funzioni di aperura e chiusura e vedo anche la barra della percentuale di posizionamento.

Mi servirebbe creare due tasti per gestire le posizioni preferite della tapparella: al 75% e al 25%.

Sto impazzendo da giorni per crearli ma non ci riesco.
Qualche suggerimento?

Grazie!

**Comment by xyzmauro** [comment]
Ciao,Potresti fare uno script che imposta al valore che vuoi la tapparella e poi con una button card lo richiami...
https://www.home-assistant.io/lovelace/button/#examples

**Comment by ilmaurino** [comment]
Grazie per il suggerimento.
guardando la schermata script in impostazioni non ho risolto molto...
Come posso richiamare un'entità? e da lì come comunico il valore?

la card ha questo codice:

type: entities
entities:
  - entity: cover.tendone
    name: Tenda sala
    icon: mdi:storefront
  - type: custom:slider-entity-row
    entity: cover.tendone
    full_row: true
    step: 1

Scusate, ma sto ancora tentando di capire come "ragiona" Home Assistant e certe volte mi sembra di perdermi in un bicchiere d'acqua...

**Reply by xyzmauro** [reply]
Devi proprio scriverti lo script che ti serve e poi dalla button card lo richiami
https://www.home-assistant.io/integrations/script/
